packages/distribution/addon/components/cd-inquiry-edit-form.js

```diff
@@ -51,4 +55,12 @@ export default class CdInquiryEditFormComponent extends Component {
       );
     }
   }
+
+  async willDestroy() {
+    super.willDestroy();
+
+    await this.apollo.apolloClient.refetchQueries({
+      include: [inquiryEditQuery],
```

Comment:
Shouldn't we refetch the dialog query?

Reply:
That works too, but then I refetch all of the inquiries on the dialog, when I only want the one inquiry. In addition, the inquiryDialogQuery seems to usually have multiple active observable queries, which then refetches all of them. Though I can ensure that only the dialog query of the current addressed / controlling group is fetched through the use of  onQueryUpdated.

Reply:
Let's just fire a query that fetches exactly what we need.. We don't really need to use "refetching".

Reply:
Ok, I'll rewrite it :+1: